hammer_cli_foreman_leapp.gemspec

```diff
@@ -22,8 +22,6 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.test_files    = Dir['{test}/**/*']
 
-  spec.add_dependency 'hammer_cli_foreman', '>= 2.0.0', '< 4.0.0'
-  spec.add_development_dependency 'bundler'
-  spec.add_development_dependency 'rake'
-  spec.add_development_dependency 'rubocop', '~> 0.80'
+  spec.add_dependency 'hammer_cli_foreman', '~> 3.9'
```

Comment:
This is pretty tight. It wouldn't work in nightly where it's now 3.11. Any particular reason to make it so tight?

Reply:
Oh, I just realized that this will work. Disregard my comment.